tasks/install_system.yml

```diff
@@ -0,0 +1,68 @@
+- name: Make sure iscsi device for {{ vm.host }} @{{ vm.org }} is logged in
+  systemd:
+    name: vm_iscsi@{{ vm.org }}/{{ vm.host }}.service
+    state: started
+  when: vm.storage_type|default(xen_vman_storage_type) == "iscsi"
+
+- name: Format root storage for {{ vm.host }} @{{ vm.org }}
+  shell: mkfs.{{ vm.filesystem|default(xen_vman_default_filesystem) }} "/dev/disk/by-path/ip-{{ xen_vman_iscsi_server }}-iscsi-{{ xen_vman_iscsi_base_wwn }}:{{ vm.org }}-{{ vm.host }}-lun-0"
+  when: vm.storage_type|default(xen_vman_storage_type) == "iscsi"
+
+- name: Make sure vm root is mounted
+  systemd:
+    name: vm_mount@{{ vm.org }}/{{ vm.host }}.service
+    state: started
+
+# TODO: make system installation async
+- name: Install {{ vm.os.type }} {{ vm.os.version }} at {{ vm.host }} @{{ vm.org }}
+  shell: debootstrap --include openssh-server,python {{ vm.os.version }} /mnt/vms/{{ vm.org }}-{{ vm.host }}
```

Comment:
Did you acutally install deboostrap? And why not softcode those default packages? And what about switching to minbase?